DemoqaProject/pageObjects/Elements.cs

```diff
@@ -7,10 +7,323 @@ namespace DemoqaProject.PageObjects
+        
+        [FindsBy(How = How.XPath, Using = "//label[@for='noRadio']")]
+        public IWebElement noRadio;
+        
+        [FindsBy(How = How.ClassName, Using = "mt-3")]
+        public IWebElement checkState;
+        
+        [FindsBy(How = How.XPath, Using = "//span[text()='Web Tables']")]
+        public IWebElement WebTablesButton;
+        
+        [FindsBy(How = How.Id, Using = "addNewRecordButton")]
+        public IWebElement AddButton;
+        
+        [FindsBy(How = How.Id, Using = "firstName")]
+        public IWebElement firstNameInput;
+        
+        [FindsBy(How = How.Id, Using = "lastName")]
+        public IWebElement lastNameInput;
+        
+        [FindsBy(How = How.Id, Using = "userEmail")]
+        public IWebElement userEmailInput;
+        
+        [FindsBy(How = How.Id, Using = "age")]
+        public IWebElement ageInput;
+        
+        [FindsBy(How = How.Id, Using = "salary")]
+        public IWebElement salaryInput;
+        
+        [FindsBy(How = How.Id, Using = "department")]
+        public IWebElement departmentInput;
+        
+        [FindsBy(How = How.Id, Using = "submit")]
+        public IWebElement submitForm;
+        
+        [FindsBy(How = How.Id, Using = "edit-record-4")]
+        public IWebElement editRecord;
+        
+        [FindsBy(How = How.Id, Using = "delete-record-4")]
+        public IWebElement deleteRecord;
+
+        [FindsBy(How = How.XPath, Using = "//div[@class='rt-td']")]
+        public IWebElement checkFirstname;
+        
+        [FindsBy(How = How.XPath, Using = "//option[@value='5']")]
+        public IWebElement valueRows;
+        
+        [FindsBy(How = How.XPath, Using = "//select[@aria-label='rows per page']")]
+        public IWebElement rowChangeto5;
+        
+        [FindsBy(How = How.XPath, Using = "//*[text()='Previous']")]
+        public IWebElement previousButton;
+        
+        [FindsBy(How = How.XPath, Using = "//*[text()='Next']")]
+        public IWebElement nextButton;
+        
+        [FindsBy(How = How.ClassName, Using = "-totalPages")]
+        public IWebElement totalPages;
+        
+        [FindsBy(How = How.Id, Using = "searchBox")]
+        public IWebElement searchBox;
+        
+
         public Elements()
         {
             PageFactory.InitElements(driver, this);
         }
 
+        public void NavigateToTextBox(string fullname, string email, string currentAddress, string permanentAddress)
+        {
+            textBoxButton.Click();
+            fullNameInput.SendKeys(fullname);
+            emailInput.SendKeys(email);
+            currentAddressInput.SendKeys(currentAddress);
+            permanentAddressInput.SendKeys(permanentAddress);
+            submitButton.Click();
+            
+            IJavaScriptExecutor js=driver as IJavaScriptExecutor;
+            js.ExecuteScript("userForm.reset()");
+            
+        }
+
+        //Getting text of elements
+        public string FullNameText()
```

Comment:
small tip: you can use such construction: public string FullNameText() => fullnameText.Text;
It's saves some lines and easy to read.

Reply:
done